src/core/ext/client_config/lb_policy_factory.h

```diff
@@ -47,8 +47,16 @@ struct grpc_lb_policy_factory {
   const grpc_lb_policy_factory_vtable *vtable;
 };
 
+typedef struct grpc_lb_policy_address_token {
+  uint8_t *token;
+  size_t token_size;
+} grpc_lb_policy_address_token;
+
 typedef struct grpc_lb_policy_args {
   grpc_resolved_addresses *addresses;
```

Comment:
We've got a few bits that need to be transported through here.

@markdroth WDYT about changing `grpc_resolved_addresses *` here and having `grpc_lb_address *`:

```c
struct grpc_lb_address {
  grpc_resolved_address addr;
  bool is_balancer;
  gpr_slice token;
};
```